source_code/main_mcu/src/COMMS/comms_aux_mcu.c

```diff
@@ -431,51 +537,51 @@ comms_msg_rcvd_te comms_aux_mcu_routine(msg_restrict_type_te answer_restrict_typ
     else if (aux_mcu_receive_message.message_type == AUX_MCU_MSG_TYPE_AUX_MCU_EVENT)
     {
         msg_rcvd = EVENT_MSG_RCVD;
-        
+
         /* Call dedicated function */
         comms_aux_mcu_deal_with_received_event(&aux_mcu_receive_message);
-    }  
+    }
     else if (aux_mcu_receive_message.message_type == AUX_MCU_MSG_TYPE_RNG_TRANSFER)
     {
         msg_rcvd = RNG_MSG_RCVD;
-        
+
         /* Set same message type and fill with random numbers */
         aux_mcu_send_message.message_type = aux_mcu_receive_message.message_type;
         rng_fill_array(aux_mcu_send_message.payload, 32);
         aux_mcu_send_message.payload_length1 = 32;
-        
+
         /* Send message */
         comms_aux_mcu_send_message(FALSE);
     }
-    else
+    else if (aux_mcu_receive_message.message_type == AUX_MCU_MSG_TYPE_FIDO2)
     {
-        msg_rcvd = UNKNOW_MSG_RCVD;
-        asm("Nop");        
-    } 
```

Comment:
actually could you keep the else{} as well?

Reply:
yes